spec/unleash/toggle_fetcher_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "spec_helper"
+
+RSpec.describe Unleash::ToggleFetcher do
+  describe '#save!' do
+    context 'when toggle_cache generation fails' do
+      let(:toggle_fetcher) { Unleash::ToggleFetcher.new }
+
+      before do
+        allow(toggle_fetcher).to receive(:toggle_cache).and_raise(StandardError)
+      end
+
+      it 'closes the file properly' do
+        toggle_fetcher.save!
+
+        backup_file = Unleash.configuration.backup_file
+        backup_file_tmp = "#{backup_file}.tmp"
+
+        backup_files = ObjectSpace.each_object(File).select { |f| f.path == backup_file_tmp }
+        
```

Comment:
There's a tiny bit of trailing white space here that Rubocop seems to not like

```suggestion

```

Reply:
Sharp eyes here. Thanks for the comment.